src/OpenLoco/Windows/PromptOkCancelWindow.cpp

```diff
@@ -6,8 +6,21 @@
 #include "../OpenLoco.h"
 #include "../Ui.h"
 #include "../Ui/WindowManager.h"
+#include "../Win32.h"
 #include <cstring>
 
+#ifdef _WIN32
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif
+#define WIN32_LEAN_AND_MEAN
+#include <shlobj.h>
+#include <windows.h>
```

Comment:
do we really need to include all of this just for  `VK_ESCAPE`

Reply:
I don't like it either; the same thing happens in TextInputWindow. As I said above, I'd like us to move to the SDL2 constants for these at some point.

Alternatively, I suppose we could start using our stub `Win32.h` file for Windows as well? If `windows.h` does get included, that should just redefine the constants and not be in the way.

Reply:
I've switched it to work with an SDL2 constant instead. Was surprisingly easy; works on Linux at least.